zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -445,11 +445,11 @@ public void runAll() {
         AuthenticationInfo authenticationInfo = new AuthenticationInfo();
         authenticationInfo.setUser(cronExecutingUser);
         p.setAuthenticationInfo(authenticationInfo);
-
-        p.setListener(jobListenerFactory.getParagraphJobListener(this));
-        Interpreter intp = factory.getInterpreter(getId(), p.getRequiredReplName());
-
-        intp.getScheduler().submit(p);
+        try {
+          run(p.getId());
+        } catch (InterpreterException intpException) {
+          throw intpException;
+        }
```

Comment:
Yes, basically, we either better either remove `try..catch` completely or refactor it to catch more specific exception + document why there is nothing happening in catch. See [Zeppelin's Java styleguide](https://google.github.io/styleguide/javaguide.html#s6.2-caught-exceptions) for more detailed explanation.